app/src/main/java/edu/cascadia/mobas/pollinatorpathway/ui/login/PlantingDao.java

```diff
@@ -0,0 +1,35 @@
+package edu.cascadia.mobas.pollinatorpathway.ui.login;
+
+import androidx.room.Dao;
+import androidx.room.Delete;
+import androidx.room.Insert;
+import androidx.room.OnConflictStrategy;
+import androidx.room.Query;
+import androidx.room.Update;
+import java.util.List;
+
+
+@Dao
+public interface PlantingDao {
+
+    @Query("SELECT * FROM Planting WHERE plantingId = :plantingId")
+    public Planting getPlanting(long plantingId);
```

Comment:
@fabricekalvanda Would you please add an issue to add the new data layer architectural elements, LiveData, Repository, Dependency Injection and Thread Executors throughout. It would be good to get this work started sooner rather than after people start coding too much data into Views